src/console/dbseed/db-seed.module.ts

```diff
@@ -20,7 +24,16 @@ import { ServiceProviderModule } from '../../modules/service-provider/service-pr
         ServiceProviderModule,
         LoggerModule.register(DbSeedModule.name),
     ],
-    providers: [DbSeedService, DbSeedMapper, DbSeedConsole, DBiamPersonenkontextRepo],
+    providers: [
+        DbSeedService,
+        DbSeedMapper,
+        DbSeedConsole,
+        DBiamPersonenkontextRepo,
+        KeycloakUserService,
+        KeycloakAdministrationService,
+        KeycloakAdminClient,
+        KeycloakInstanceConfig.fromConfigService(),
```

Comment:
Instead of providing manually, just add the `KeycloakAdministrationModule` to the `imports`-Array